src/analysis/retail/hunter/marksmanship/modules/talents/AimedShot.tsx

```diff
@@ -77,9 +88,23 @@ class AimedShot extends Analyzer {
      * Dead Eye and Trueshot scale multiplicatively off each other, which can lead to extremely fast cooldown reduction that this should properly handle.
      */
     let modRate = 1;
+
+    // Sum the total cdr reduction from Trueshot, Tactial Reload, Steady Focus
+    let reduction = 0;
     if (this.selectedCombatant.hasBuff(SPELLS.TRUESHOT.id)) {
-      modRate /= 1 + TRUESHOT_AIMED_SHOT_RECHARGE_INCREASE;
+      reduction += TRUESHOT_AIMED_SHOT_RECHARGE_INCREASE;
+    }
+
+    if (this.selectedCombatant.hasTalent(TALENTS_HUNTER.TACTICAL_RELOAD_TALENT)) {
+      reduction += TACTICAL_RELOAD_CDR_REDUCTION;
+    }
+
+    if (this.selectedCombatant.hasBuff(SPELLS.STEADY_FOCUS_BUFF.id)) {
+      reduction += STEADY_FOCUS_HASTE_PERCENT[1];
```

Comment:
You can just add the steady focus haste to the haste file and it will automatically be handled in the abilities file since it's already hasted cooldown in there.

Reply:
There is a file called Haste in src/parser/shared/modules that already has Steady Focus buff defined as a 7% buff - you can update it to 8%, but shouldn't need any handling inside the Aimed Shot file.

Reply:
Good call out. Thanks for pointing this out. Changed the value and reverted my changes in Aimed Shot.

Follow up question, is it safe to assume any buffs/talents that contributes a static haste % should live in this haste file?